wsagent/che-core-api-git-shared/src/main/java/org/eclipse/che/api/git/shared/event/GitEvent.java

```diff
@@ -10,4 +10,7 @@
  */
 package org.eclipse.che.api.git.shared.event;
 
-public interface GitEvent {}
+public interface GitEvent {
+  /** Returns name of project in which the event was fired. */
+  String getProjectName();
```

Comment:
OK to me, I don't see any better solution, may be it would be good to add some documentation to describe project name and that it is not a project module.